### ISSUE TITLE
gfm code blocks format error

### ISSUE THREAD

**Issue by hh02** [jgm/pandoc#6576]
Gfm can not use backtick code blocks.

## the pandoc version

pandoc --version
pandoc.exe 2.10.1
Compiled with pandoc-types 1.21, texmath 0.12.0.2, skylighting 0.8.5

## the exact command line used
`pandoc test.org -f org -t gfm -o test.md`

## the exact input used

``` org
* c code
#+BEGIN_SRC c
#include <cstdio.h>
int main() {
    return 0;
}
#+END_SRC
```
## the output received
``` md
c code
======

    #include <cstdio.h>
    int main() {
        return 0;
    }
```
## the output you expected instead

``` md
c code
======

``` c
#include <cstdio.h>
int main() {
    return 0;
}
```
```

**Comment by hh02** [comment]
This problem is solved in last commit, many thanks for your time.

**Comment by mb21** [comment]
great, than the issue can be closed...

**Comment by jgm** [comment]
For future reference, the commit that closes this is 2077739a35fd51ca6f482ce8a108d10799f5b274
